### ISSUE TITLE
Unable to print from MacOS to Debian Linux CUPS Server

### ISSUE THREAD

**Issue by michaelrsweet** [apple/cups#2771]
Version: None
CUPS.org User: myudkowsky

I'm attempting to print from a MacOS 10.4 laptop to a CUPS server running on Debian (unstable). Although the MacOS laptop can print successfully to a slightly older installation on a different CUPS server, the attempt to print to this new server (and to several new printers, a Samsung CLP-300 and a Lexmark Z42) fails.

I've tried this with three different printer drivers for the CLP-300 (the Samsung unified drivers and two open source drivers) but all fail. I've just tried again with a Lexmark Z42 -- same error. From the log file, it's clearly the GS-ESP 8.62 package that fails to render the incoming file from the Mac.

Here's the output from the Z42 attempt using the Gutenberg drivers:
D [27/Mar/2008:06:40:41 -0500] [Job 83] envp[24]="FINAL_CONTENT_TYPE=printer/Lex"
I [27/Mar/2008:06:40:41 -0500] [Job 83] Started filter /usr/lib/cups/filter/pdftops (PID 29141)
I [27/Mar/2008:06:40:41 -0500] [Job 83] Started filter /usr/lib/cups/filter/pstops (PID 29142)
I [27/Mar/2008:06:40:41 -0500] [Job 83] Started filter /usr/lib/cups/filter/pstoraster (PID 29143)
I [27/Mar/2008:06:40:41 -0500] [Job 83] Started filter /usr/lib/cups/filter/rastertogutenprint.5.0 (PID 29144)
I [27/Mar/2008:06:40:41 -0500] [Job 83] Started backend /usr/lib/cups/backend/usb (PID 29145)

Later I see

D [27/Mar/2008:06:40:41 -0500] [Job 83] Running /usr/bin/gs -dQUIET -dDEBUG -dPARANOIDSAFER -dNOPAUSE -dBATCH -dNOMEDIAATTRS -sDEVICE=cups -sstdout=%stderr -sOUTPUTFILE=%stdout -c -_

and still later

D [27/Mar/2008:06:40:42 -0500] [Job 83] Gutenprint   NOT setting CurveCh26 to (null)
D [27/Mar/2008:06:40:42 -0500] [Job 83] Error: /undefined in CF
D [27/Mar/2008:06:40:42 -0500] [Job 83] Operand stack:
D [27/Mar/2008:06:40:42 -0500] [Job 83] 
D [27/Mar/2008:06:40:42 -0500] [Job 83] Execution stack:
D [27/Mar/2008:06:40:42 -0500] [Job 83] %interp_exit   .runexec2   --nostringval--   --nostringval--   --nostringval--   2   %stopped_push   --nostringval--   --nostringval--   --nostringval--   false   1   %stopped_push   1905   1   3   %oparray_pop   1904   1   3   %oparray_pop   1888   1   3   %oparray_pop   1771   1   3   %oparray_pop   --nostringval--   %errorexec_pop   .runexec2   --nostringval--   --nostringval--   --nostringval--   2   %stopped_push   --nostringval--
D [27/Mar/2008:06:40:42 -0500] [Job 83] Dictionary stack:
D [27/Mar/2008:06:40:42 -0500] [Job 83] --dict:1152/1684(ro)(G)--   --dict:0/20(G)--   --dict:99/200(L)--   --dict:66/75(L)--   --dict:18/25(L)--
D [27/Mar/2008:06:40:42 -0500] [Job 83] Current allocation mode is local
D [27/Mar/2008:06:40:42 -0500] [Job 83] Last OS error: 2
D [27/Mar/2008:06:40:42 -0500] [Job 83] GPL Ghostscript 8.62: Unrecoverable error, exit code 1

This error was also documented in http://www.cups.org/newsgroups.php?s4881+gcups.bugs+v4890+T0, using a different printer, printer driver, and incoming message.

In short: MacOS 10.4 apparently cannot print to CUPS servers running the latest GS-ESP.

Please let me know if you need further information.
